resources/services/ec2/ebs_snapshots.go

```diff
@@ -144,6 +146,16 @@ func fetchEc2EbsSnapshots(ctx context.Context, meta schema.ClientMeta, parent *s
 	}
 	return nil
 }
+
+func IsSnapshotNotFoundError(err error) bool {
```

Comment:
We could also move this to a more central location (like the client), however I think this error is scoped only to here